.travis.yml

```diff
@@ -31,25 +29,24 @@ cache:
 
 before_install:
     - mkdir -p ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d
-    - if [[ "$TRAVIS_PHP_VERSION" == '5.5' || "$TRAVIS_PHP_VERSION" == '5.6' ]]; then ./build/5.x.sh; fi
-    - if [ "$TRAVIS_PHP_VERSION" == '7.0' ]; then ./build/7.x.sh; fi
-    - if [ "$TRAVIS_PHP_VERSION" == 'hhvm' ]; then ./build/hhvm.sh; fi
+    - if [[ $TRAVIS_PHP_VERSION = 5.* ]]; then ./build/5.x.sh; fi
+    - if [[ $TRAVIS_PHP_VERSION = 7.* ]]; then ./build/7.x.sh; fi
     - ./build/all.sh
     - travis_retry composer self-update
 
 install:
-    - COMPONENTS=$(find src/ -mindepth 3 -type f -name phpunit.xml.dist -printf '%h\n')
     - composer require --no-update "mongodb/mongodb" "^1.0"
-    - travis_retry composer install --no-interaction --prefer-source
+    - composer install --no-interaction --prefer-source
 
-script:
-    - echo "$COMPONENTS" | parallel --gnu 'printf "\n\n************ Running tests for {} ************\n\n"; vendor/bin/phpunit --exclude-group tty,benchmark,intl-data {}'
 
-after_script:
-    - if [ "$TRAVIS_PHP_VERSION" == '7.0' ]; then codecov; fi
+script:
+    - ./build/runTests.sh
```

Comment:
Yes and no.. https://getcomposer.org/doc/articles/scripts.md#writing-custom-commands

I guess the benefit is that people do not have to look for which script to run to test the package.